source/Halibut.Tests/UsageFixture.cs

```diff
@@ -257,12 +257,43 @@ public async Task OctopusCanSendAndReceiveComplexObjects_WithMultipleComplexChil
                 response.Child1.ListOfStreams.Select(x => x.ReadAsString()).ToList().Should().BeEquivalentTo(list);
                 response.Child1.DictionaryPayload.Should().NotBeSameAs(request.Child1.DictionaryPayload);
                 response.Child1.DictionaryPayload.Should().BeEquivalentTo(dictionary);
-                
+
                 response.Child2.Should().NotBeSameAs(request.Child2);
                 response.Child2.EnumPayload.Should().Be(enumValue);
                 response.Child2.ComplexPayloadSet.Should().NotBeSameAs(request.Child2.ComplexPayloadSet);
                 response.Child2.ComplexPayloadSet.Select(x => new ComplexPair<string>(x.EnumValue, x.Payload.ReadAsString())).ToHashSet().Should().BeEquivalentTo(set);
             }
         }
+
+        [Test]
+        [LatestClientAndLatestServiceTestCases(testNetworkConditions: false)]
```

Comment:
After discussion with @acodrington, we agreed that it doesn't make sense to do the backwards compatible test cases because the old versions won't have the type registry changes needed, thus they won't work.